core/controllers/topics_and_skills_dashboard.py

```diff
@@ -126,16 +126,12 @@ def get(self):
                     skills_list.append(skill_dict)
                 categorized_skills_dict[topic.name][
                     subtopic.title] = skills_list
-        categorized_skills_dict['untriaged_skills'] = []
+
         for skill_summary_dict in skill_summary_dicts:
             skill_id = skill_summary_dict['id']
             if (skill_id not in skill_ids_assigned_to_some_topic) and (
                     skill_id not in merged_skill_ids):
                 untriaged_skill_summary_dicts.append(skill_summary_dict)
-                categorized_skills_dict['untriaged_skills'].append({
-                    'skill_id': skill_id,
-                    'skill_description': skill_summary_dict['description']
-                })
```

Comment:
I removed this because of two reasons
- `categorized_skills_dict` is a dict with topic names as keys. In typescript if there's a dict all the values should be of same type.
- The same information was already being returned in `untriaged_skill_summary_dicts`.

Reply:
Thanks @nishantwrp, adding @Showtim3 for review since he should be aware of what's going on with the topics-and-skills-dashboard stuff!

Reply:
Ok @seanlip, Please note that to declare a type for this in the backend-api service. This is **necessary**.

Reply:
Yup sorry -- I don't disagree. I think @Showtim3 and you would need to collaborate to make sure that the changes to the data structure look good and are propagated correctly. Thanks!

Reply:
Yeah the changes look okay. Tested them locally also.